pkg/v1/tkg/web/src/app/views/landing/aws-wizard/aws-wizard.component.ts

```diff
@@ -332,7 +336,7 @@ export class AwsWizardComponent extends WizardBaseDirective implements OnInit {
         return this.getOsImageForm(AwsOsImageStepComponent);
     }
     get AwsNetworkForm(): FormDataForHTML {
-        return FormUtility.formOverrideDescription(this.NetworkForm, 'Specify the cluster Pod CIDR');
+        return FormUtility.formWithOverrides(this.NetworkForm, {description: 'Specify the cluster Pod CIDR'});
     }
     //
```

Comment:
Not your doing, but we could remove the comments in lines 341-342. No longer any template related convenience methods.